Server/All

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/python
+
+# Turn on debug mode.
+import cgitb
+import pymysql
+import json
+
+cgitb.enable()
+
+# Print necessary headers.
+print("Content-Type: text/html; charset=utf-8\n\n")
+
+import cgi
+form = cgi.FieldStorage()
+print ("Hello")
+print ("<br />")
+print ("<br />")
+
+#def greet_player():
+    #if "player" not in form:
+        #print ("Something Went Wrong")
+    #else:
+        #print("<b>Hello " + str(form.getvalue("player")) + "</b>")
+        #my_first_int = 1
+        #my_first_str = str(form.getvalue("user"))
+
+
+# Connect to the database.
+conn = pymysql.connect(
+    db='highscores',
+    user='root',
+    passwd='password',
+    host='localhost')
+c = conn.cursor()
+
+def greet_player():
+    if 'first_name' not in form:
+        print("Something went wrong")
+    else:
+        print("<b>Hello " + str(form.getvalue("first_name")) + "</b>")
+        
+'''
+Get information from player: first_name, last_name and score
+'''
+def get_name_score():
+    if 'first_name' and 'last_name' and 'score' not in form:
+        print ("Where are first name, last name, score?")
+    else:
+        first_name = str(form.getvalue('first_name'))
+        last_name = str(form.getvalue('last_name'))
+        score = str(form.getvalue('score'))
+        
+'''        
+I don't know why this won't work. It doesn't give an error so I have no idea what's wrong.
+'''
+def add_player(first_name, last_name):
+# Insert some example data.
+    c.execute("INSERT INTO players VALUES (NULL, ' + first_name + ', ' + last_name + ')")
```

Comment:
I think it's a problem with where you have used single/double quotes...
"INSERT INTO players VALUES (NULL, ' " + first_name + "," + last_name + " ')"